core/src/logging/filter/mod.rs

```diff
@@ -352,26 +402,26 @@ impl LogIdModuleFilter {
 }
 
 #[derive(Default, Debug)]
-pub(super) struct InnerLogFilter {
-    no_general_logging: bool,
+pub struct FilterConfig {
+    logging_enabled: bool,
```

Comment:
```suggestion
    general_logging_enabled: bool,
```

Because this setting only affects global log level and IDs, but not module specific settings.